src/test/java/com/google/sps/data/DatastoreScheduledInterviewDaoTest.java

```diff
@@ -190,6 +190,94 @@ public void getForPerson() {
     Assert.assertEquals(expected, result);
   }
 
+  // Tests that only interviews without a shadow are returned.
+  @Test
```

Comment:
lf you add the other three filters as specified in my first comment make sure to add tests here

Reply:
Tried it and found that it was not possible (sadly).